modules/i18n/src/translations/en-US/portals/applications.ts

```diff
@@ -1629,6 +1629,10 @@ export const applications: ApplicationsNS = {
                             label: "Client authentication method",
                             placeholder: "Select method"
                         },
+                        reusePvtKeyJWT: {
+                            hint: "By default private key JWT is not allowed to reuse. But if you have a requirment to reuse it. Please click on the checkbox.",
```

Comment:
It'd be better to get this text reviewed with the doc and ux teams. 

cc: @Achintha444 @savindi7 @himeshsiriwardana 

The following is just a suggestion.

```suggestion
                            hint: "Whether the private key JWT can be used multiple times. By default, private key JWTs are not allowed to be reused.",
```

Reply:
Updated the content with the previously approved org level config.

<img width="826" alt="Screenshot 2024-07-02 at 07 05 48" src="https://github.com/wso2/identity-apps/assets/47152272/d0e6b003-489a-4357-936c-35de40e6c77f">

The new UI will look like this. Changed the word `Token` to `Private Key JWT` as Token is too generic and can be confused with the Access Token.

<img width="867" alt="Screenshot 2024-07-02 at 07 06 13" src="https://github.com/wso2/identity-apps/assets/47152272/f0cbad06-e3aa-4052-8adc-aef2f653101c">

Reply:
@savindi7 @Achintha444 Please review. Will update the mail thread once this is approved.

Reply:
+1 for the current text updated text.